python/python/bystro/api/cli.py

```diff
@@ -572,6 +626,14 @@ def main():
     )
     jobs_parser.set_defaults(func=get_jobs)
 
+    query_parser = subparsers.add_parser("query", help="Perform a query")
+    query_parser.add_argument("--dir", default=DEFAULT_DIR, help="Where Bystro API login state is saved")
+    query_parser.add_argument("--query", required=True, help="Query properties")
```

Comment:
Let's change the help for query to "The OpenSearch query string query, e.g. (cadd: >= 20)"